Tests/DependencyInjection/AbstractIvoryCKEditorExtensionTest.php

```diff
@@ -331,6 +338,26 @@ public function testCustomPaths()
         $this->assertSame('foo/ckeditor.js', $ckEditorType->getJsPath());
     }
 
+    public function testTemplatingConfiguration()
+    {
+        // remove mock definition from container builder before compiling
+        $this->container->compile();
+
+        $services = array(
+            'ivory_ck_editor.templating.helper',
+        );
+
+        foreach ($services as $service) {
+            $has = $this->container->hasDefinition($service);
+            $this->assertTrue($has, sprintf('Missing service definition for "%s"', $service));
+            if ($has) {
+                $def = $this->container->getDefinition($service);
+                $ref = $def->getArgument(0);
+                $this->assertSame('service_container', (string) $ref);
```

Comment:
Instead of checking the definition, can't you just fetch the service? If an error occured, then there is something wrong. Checking we inject the container will be done with the constructor typehint.